src/wp-admin/includes/ajax-actions.php

```diff
@@ -330,6 +330,17 @@ function wp_ajax_autocomplete_user() {
 	wp_die( wp_json_encode( $return ) );
 }
 
+
+/**
+ * Handles Ajax requests for clearing community events
+ */
+function wp_ajax_clear_community_events() {
+	check_ajax_referer( 'community_events' );
+	$user_id        = get_current_user_id();
+	update_user_option( $user_id, 'community-events-location', null, true );
```

Comment:
It feels a bit more clear to me to delete the option rather than setting it to `null`.